webapp/carts/models.py

```diff
@@ -0,0 +1,42 @@
+from django.conf import settings
+from django.db import models
+
+
+User = settings.AUTH_USER_MODEL
+
+
+class Cart(models.Model):
+    #allowed nulls and blanks for future guest checkout
+    user = models.ForeignKey(
+        User,
+        null=True,
+        blank=True)
+
+    book = models.ForeignKey(
```

Comment:
The cart should support multiple books with varying quantities.
Perhaps you should have a sub-model to keep track of individual item orders?

Reply:
This will be a many to many relationship later on.

Reply:
Why not now? Not sure what the benefit is here; if it were incomplete I'd agree, but it's currently inaccurate, so we should not have it in our master branch.